src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/resource/swagger/AuthenticatedApiResponses.kt

```diff
@@ -0,0 +1,30 @@
+package uk.gov.justice.digital.hmpps.manageusersapi.resource.swagger
```

Comment:
This class if for those endpoints where no role is required (i.e. just need to be authenticated)

Reply:
very neat implementation